gap/oper.gi

```diff
@@ -1338,6 +1360,7 @@ function(digraph)
   od;
   gr := DigraphNC(new_adj, tot);
   SetDigraphVertexLabels(gr, DigraphVertexLabels(digraph));
+  # Multidigraphs did not have edge labels
```

Comment:
It could be the case that `DigraphRemoveAllMultipleEdges` is called on a digraph which has no multiple edges -- in this case we'd probably just want to return the original digraph. But I don't expect you to make that change.